### PR TITLE
serialization: rebase on docker draft

### PR DESCRIPTION
**Pull request by vbatts**
from https://github.com/docker/docker/pull/22264
specifically commit a0237314277148093987380edc10222e63ba19e2

See also https://github.com/opencontainers/image-spec/issues/36#issuecomment-215175217

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

**Comment by philips** [comment]
LGTM

**Comment by vbatts** [comment]
rebased to include https://github.com/opencontainers/image-spec/commit/949ea01d9d0f0a42d3a5f864ed6adacdeac0b826

@timthelion comments on https://github.com/docker/docker/pull/22264#discussion_r61455320 ?

**Comment by philips** [comment]
ping @timthelion I would like to get this closed out for v0.1.0.

File: serialization.md [hunk 1]
```diff
@@ -119,31 +121,35 @@ Here is an example image JSON file:
...
 ```
 
+Note that image JSON files produced by Docker don't contain formatting whitespace.
+It has been added to this example for clarity.
+
 ### Image JSON Field Descriptions
 
 <dl>
     <dt>
-        id <code>string</code>
```
Comment (stevvooe) [inline serialization.md]: Why is this getting dropped?

**Pull request merged**